gensrc/thrift/Planner.thrift

```diff
@@ -65,6 +66,13 @@ struct TPlanFragment {
   // sink) in a single instance of this fragment. This is used for an optimization in
   // InitialReservation. Measured in bytes. required in V1
   8: optional i64 initial_reservation_total_claims
+  20: optional list<TGlobalDict> global_dicts
+}
+
+struct TGlobalDict {
```

Comment:
we defined it in Data.thrift, we could remove it

Reply:
Yes. I thought I removed it...